data/json/snippets/grave_labels.json

```diff
@@ -31,7 +32,13 @@
       "I held you your whole life",
       "So small, so sweet, so soon",
       "Ours for a little while",
-      "You died for us, we'll live for you"
+      "You died for us, we'll live for you",
+      "Sorry, you had it coming",
+      "<nothing is written here>",
+      "Didn't have to end like this",
+      "Should've left your corpse where it lay, but I'm afraid it'd rise again and I'd have to put you down again",
+      "I wish you could've seen reason",
+      "A lesser man would've let you turn"
```

Comment:
The meta knowledge in this one has me shook! I love it, someone had in genre knowledge that zombie bites infect the living and turn them into zombies but that's not how zombies work in this world.  I adore this snippet for that.